lib/boruta/config.ex

```diff
@@ -135,12 +135,17 @@ defmodule Boruta.Config do
     Keyword.fetch!(oauth_config(), :issuer)
   end
 
+  @doc false
+  def redirect_uri_validation_fun do
+    Keyword.get(override_config(), :redirect_uri_validation_fun)
```

Comment:
I think the best would be to set a default function which can be defined in the `lib/boruta/utils` folder, that would prevent the deepen case in the validation. WDYT?

Reply:
Not sure I understand the suggestion, but yeah whatever you think is best.

Reply:
What I have in mind is to provide a default implementation of the function that would be registered in `@default`in the file that would be overridable in custom config.

Reply:
Gotcha. What I did was extending the function (since valid URIs should also be accepted and I didn't want to re-implement that) but that's fine too.

Reply:
Long time no news, do you want me to implement the changes?

Reply:
Thanks for the ping, I may look at it today

Reply:
alright, it's a bit more verbose but I guess clearer?